internals/scripts/validate-config.js

```diff
@@ -1,5 +1,5 @@
 const { validator } = require('@exodus/schemasafe');
-const schema = require('../../environment.conf.schema.json');
+const schema = require('../../internals/schemas/environment.conf.schema.json');
```

Comment:
This could be more simple:

    const schema = require('../schemas/environment.conf.schema.json');

Reply:
@boris-arkenaar I propose to even simplify it more and use:
```
npx ajv-cli validate -s internals/schemas/environment.conf.schema.json -d environment.conf.json
``

Which eliminates the need of the custom script, do you agree?

Reply:
Might work. Would be nice if this removes the need for the script. Needs to be tested though, also from `start.sh` in the container.

Reply:
Thanks for your thoughts, I am not worried about if it will work since this seems the unwritten `standard` for the `js` world to validate schemas and works fine in my tests.

This tool came up while I am doing investigation for another ticket and is (for me) out of scope of this pull request, thanks!